server/src/lsp-server.ts

```diff
@@ -80,9 +80,11 @@ export class LspServer {
             return executable;
         }
         // 2) use globally installed tsserver
-        if (commandExists.sync(getTsserverExecutable())) {
-            return getTsserverExecutable();
-        }
+        try {
+          let path = require.resolve(getTsserverExecutable())
+          return path
+        } catch (e) {}
```

Comment:
Before just `tsserver` was being executed since we had the ability to execute files (which would start with an arbitrary node server, whatever the hashbang resolved to). Now we resolve the package `typescript/bin/tsserver` to get the path on disk.